community/modules/scheduler/htcondor-configure/README.md

```diff
@@ -72,9 +72,81 @@ example can be found in the [examples README][htc-example].
   - internal_ip
 ```
 
-A full example can be found in the [examples README][htc-example].
+## High Availability
 
-[htc-example]: ../../../../examples/README.md#htcondor-poolyaml--
+This module supports high availability modes of the HTCondor Central Manager and
+of the Access Points. In these modes, the services can be resiliant against
+zonal failures by distributing the services across two zones. To implement
+HA of the Central Manager service, apply the startup script in the example
+snipppet to 2 VMs. The 2 VMs should differ only by using the primary and
+secondary IP addresses created by this module.
```

Comment:
Just from naive inspection it would seem both of these VMs would inherit the same `zone` from top level vars.  This seems counter to being resilient to "zonal failures". What am I missing here?